src/main/resources/assets/forge/lang/en_us.json

```diff
@@ -156,6 +156,8 @@
   "forge.controlsgui.alt": "ALT + %s",
 
   "forge.container.enchant.limitedEnchantability": "Limited Enchantability",
-  "attribute.name.generic.reachDistance": "Reach Distance",
-  "attribute.name.forge.swimSpeed": "Swim Speed"
+  "attribute.name.forge.swim_speed": "Swim Speed",
+  "attribute.name.forge.nametag_distance": "Nametag Render Distance",
+  "attribute.name.forge.entity_gravity": "Entity Gravity",
+  "attribute.name.generic.reach_distance": "Reach Distance"
```

Comment:
Is the naming change *needed* for it to work, or can it work with the old naming? The reason I ask is if memory serves if the keys get renamed stuff needs to happen in crowdin/split as forge currently is using the same lang instance on crowdin between 1.15 and 1.16 (and I think maybe even 1.14)

Reply:
It does work with the old naming; my reasoning for changing it is to make it consistent with how vanilla names the translation keys for their attributes. If you wish, I can use the old naming (_though I really wish we can make it consistent with vanilla_).

Reply:
@sciwhiz12 You will probably need both entries in there. Since the data is indeed pulled for both 1.15 and 1.16 so this would be a breaking change in both branches.

Reply:
So I've added both translation entries for the attributes to the en_us.json file, but the code only uses the old translation keys for now. This should prevent this from being a breaking change now.